### PR TITLE
[tests-only][full-ci]Remove spaces tests with /shares on path from apiShareOperationsToShares2/uploadToShare

### PR DESCRIPTION
**Pull request by SwikritiT**
## Related Issue

- part of: https://github.com/owncloud/ocis/issues/4154

## How Has This Been Tested?

- local
- CI
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiShareOperationsToShares2/uploadToShare.feature [hunk 1]
```diff
@@ -193,11 +168,7 @@ Feature: sharing
       | old      |
       | new      |
 
-    @skipOnOcV10 @personalSpace
-    Examples:
-      | dav-path |
-      | spaces   |
-
+
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiShareOperationsToShares2/uploadToShare.feature:171]: do we need this?

File: tests/acceptance/features/apiShareOperationsToShares2/uploadToShare.feature [hunk 2]
```diff
@@ -297,11 +253,7 @@ Feature: sharing
...
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiShareOperationsToShares2/uploadToShare.feature:256]: same here. 
others lgtm

**Review by phil-davis — APPROVED** [review]

**Review by SagarGi — COMMENTED** [review]
LGTM

**Comment by SwikritiT** [comment]
I'm running tests from this branch in https://github.com/owncloud/ocis/pull/4371, I'll merge this PR after both PR's are green

**Review by kiranparajuli589 — APPROVED** [review]

**Comment by SwikritiT** [comment]
I've made changes again in https://github.com/owncloud/core/pull/40276/commits/3cade2050190491e1a224bbecb15ef41120328d5 Please review again

File: tests/TestHelpers/WebDavHelper.php [hunk 3]
```diff
@@ -113,6 +113,49 @@ public static function getFileIdForPath(
...
+	
+	/**
+	 * returns body for propfind
+	 *
+	 * @param array|null $properties
+	 *
+	 * @return string
+	 * @throws Exception
+	 */
+	public static function getBodyForPropfind($properties) {
```
Comment (phil-davis) [inline tests/TestHelpers/WebDavHelper.php:125]:
```suggestion
	public static function getBodyForPropfind(?array $properties):  string {
```

IMO we can declare the types these days.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**